packages/mobile/src/screens/chat-screen/ChatTextInput.tsx

```diff
@@ -26,17 +27,26 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
     alignItems: 'flex-end',
     backgroundColor: palette.neutralLight10,
     paddingLeft: spacing(4),
+    paddingVertical: spacing(1),
     borderRadius: spacing(1)
   },
   composeTextInput: {
     fontSize: typography.fontSize.medium,
     lineHeight: spacing(6),
+    justifyContent: 'center',
+    alignItems: 'center',
     paddingTop: 0
   },
   icon: {
-    width: spacing(7),
-    height: spacing(7),
-    fill: palette.primary
+    width: spacing(5),
+    height: spacing(5),
+    fill: palette.white
+  },
+  iconCircle: {
+    borderRadius: spacing(5),
+    paddingVertical: spacing(1.5),
+    paddingLeft: 5,
```

Comment:
nit: odd numbers a bit weird

Reply:
Yeahhhh I know but this is how it was in the design :(